syft/core/hooks/base.py

```diff
@@ -1,4 +1,6 @@
-from abc import ABCMeta, abstractmethod
+from abc import ABCMeta
+from abc import abstractmethod
```

Comment:
Cool, in that case, the file mentioned above should be changed as well.

Reply:
Oh, actually pep8 says otherwise:
```
Imports should usually be on separate lines:

Yes: import os
     import sys

No:  import sys, os

It's okay to say this though:

from subprocess import Popen, PIPE
```

Since we're importing these from the same module, looks like we're fine with how it was before.

Reply:
Will fix. My import fixer hook was having issues so I will try to debug and add back in the future

Reply:
Separate lines is better for merge conflicts though

Reply:
If my only merge conflicts are in import statements, I'm a happy man :)  Happy to hear what the others think about it.

Also worth noting that TF/PyTorch/most open-source python projects I've seen do one line per module.

Reply:
I don't have a problem with changing it back would love to hear other opinions